NicolaeChristof/Rise
# Added the new music tracks, as well as a new variable to ensure that …

This PR adds the new spring and autumn music tracks, and fixes an issue where clicking the restart button would just play the main menu track.

How to Test:
Play around with the different levels (start your game up from different scenes, even) to ensure that the right tracks play. I didn't test out if the right music plays when you hit restart after running out of time.
